web/api/web_api_v1.c

```diff
@@ -874,10 +874,13 @@ inline int web_client_api_request_v1_info_fill_buffer(RRDHOST *host, BUFFER *wb)
 #else
     buffer_strcat(wb, "\t\"cloud-available\": false,\n");
 #endif
-    if (is_agent_claimed() == NULL)
+    char *agent_id = is_agent_claimed();
+    if (agent_id == NULL)
         buffer_strcat(wb, "\t\"agent-claimed\": false,\n");
```

Comment:
this is not the fault of this PR but I think we forgot to update the swagger docs with `agent-*` vars

Reply:
In theory...